lib/urcli-money.js

```diff
@@ -66,8 +74,16 @@ function definePeriods () {
       }
       const yearAndMonth = arg < 9 ? `${year}-0${arg}` : `${year}-${arg}`
       let start = moment(yearAndMonth).utc()
-      let end = moment(yearAndMonth).utc().add(1, 'M')
-      periods.push([start, end])
+      // Check if month asked is the current month
+      if (arg === String(month)) {
```

Comment:
Since `String()` actually calls `toString()` internally, and `toString()` throws if the value is `undefined` or `null`, I think `toString()` should be preferred.